src/redux/configureStore.jsx

```diff
@@ -0,0 +1,13 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+import bookStatusReducer from './categories/categories';
+
```

Comment:
Great job @Ademola-coding 🤝

- [ ] As stated in the project requirements, kindly set the initial state to be an empty array of categories.

Reply:
DONE